user/serializers.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_stuff")
```

Comment:
There is a typo in the `read_only_fields` attribute. It should be `is_staff` instead of `is_stuff`. This typo will cause an error when attempting to use the serializer.